GameData/RealismOverhaul/RO_SuggestedMods/BahamutoD/RO_BahamutoD_Constellation.cfg

```diff
@@ -452,88 +385,55 @@
     @node_attach = 0.0, 2.615, 0.0, 0.0, 1.0, 0.0
 
     @category = Engine
-    @title = RS-68 Series
-    @manufacturer = Aerojet Rocketdyne
-    @description = The RS-68 is gas generator engine design, powered by liquid hydrogen and liquid oxygen. It is used on the Common Booster Core (CBC) of the Delta IV launch vehicle. 
 
     @mass = 6.6
     @crashTolerance = 10
     @breakingForce = 250
     @breakingTorque = 250
-    @maxTemp = 1023.15
-    %skinMaxTemp = 1773.15
-    %heatConductivity = 0.06
-    %skinInternalConductionMult = 4.0
-    %emissiveConstant = 0.8
+    @maxTemp = 573.15
+    %skinMaxTemp = 673.15
     %bulkheadProfiles = size2
 
     %engineType = RS68
     %engineTypeMult = 1
     %massOffset = 0
     %ignoreMass = False
 
-    !MODULE,*{}
-
-    MODULE
+    @MODULE[ModuleEngines*]
     {
-        name = ModuleEnginesRF
-        thrustVectorTransformName = thrustTransform
-        exhaustDamage = True
-        ignitionThreshold = 0.1
-        minThrust = 1890
-        maxThrust = 3370
-        heatProduction = 100
-        exhaustDamageMultiplier = 1.0
-        EngineType = LiquidFuel
-
-        PROPELLANT
+        @name = ModuleEnginesRF
+        @minThrust = 1890
+        @maxThrust = 3370
+        @heatProduction = 100
+
+        @PROPELLANT[LiquidFuel]
         {
-            name = LqdHydrogen
-            ratio = 0.729
-            DrawGauge = True
+            @name = LqdHydrogen
+            @ratio = 0.7285
         }
 
-        PROPELLANT
+        @PROPELLANT[Oxidizer]
         {
-            name = LqdOxygen
-            ratio = 0.271
-            DrawGauge = False
+            @name = LqdOxygen
+            @ratio = 0.2715
         }
 
-        atmosphereCurve
+        @atmosphereCurve
         {
-            key = 0 414
-            key = 1 366
+            @key,0 = 0 409
+            @key,1 = 1 357
         }
     }
 
-    MODULE
+    @MODULE[SmarterGimbal]
```

Comment:
Someone, someone did not a perfect job at updating it. This module comes from the "Exsurgent Engineering" mod that has not been updated for at least 2 years. I consider it deprecated since the stock gimbal module includes all the mod fixes and enhancements that were available over the years.